addons/test_mail/tests/test_mail_composer.py

```diff
@@ -430,6 +442,40 @@ def test_mail_composer_rights_portal(self):
             self.assertEqual(self.test_record.message_ids[0].body, '<p>Body text 2</p>')
             self.assertEqual(self.test_record.message_ids[0].author_id, portal_user.partner_id)
 
+    @users('user_rendering_restricted')
+    def test_mail_composer_rights_attachments(self):
```

Comment:
Would define it before test_mail_composer_rights_portal (because tests are launched alphabetically, so that crashes follow the file order)